DomainContent/Hub/pill/scripts/swallowFXPill.js

```diff
@@ -115,7 +115,8 @@
         checkIfNearHead: function() {
             if (_this.isInactive && HMD.active) {
                 var position = Entities.getEntityProperties(_entityID, "position").position;
-                if (Vec3.distance(position, MyAvatar.getJointPosition("Head")) < (CHECK_RADIUS * MyAvatar.scale)) {
+                if ((Vec3.distance(position, MyAvatar.getJointPosition("Head")) < (CHECK_RADIUS * MyAvatar.scale)) || 
```

Comment:
Extra parenthesis not needed for two comparisons around the || operator.   Suggestion to calculate and store CHECK_RADIUS * MyAvatar.scale in variable before using it twice.